pt-BR/mail/chrome/messenger/AccountManager.dtd

```diff
@@ -38,6 +38,6 @@
 <!ENTITY pbAccountTree.width "width: 25ch;">
 
 <!ENTITY pbAccountAddActionsButton.accesskey "A">
-<!ENTITY pbAccountAddActionsButton.label "Add…">
-<!ENTITY pbAccountRemoveActionsButton.label "Remove">
-<!ENTITY pbAccountRemoveActionsButton.accesskey "R">
\ No newline at end of file
+<!ENTITY pbAccountAddActionsButton.label "Adicionar…">
+<!ENTITY pbAccountRemoveActionsButton.label "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbAccountRemoveActionsButton.label "Excluir">